src/js/index.js

```diff
@@ -70,45 +78,51 @@ export const hpe = deepFreeze({
         light: '#999999',
       },
       'selected-background': 'brand',
-      'selected-text': '#FFFFFF',
-      'status-critical': '#FF4040',
-      'status-warning': '#FFAA15',
-      'status-ok': '#00C781',
-      'status-unknown': '#CCCCCC',
+      'selected-text': 'text-strong',
+      'status-critical': {
+        dark: 'red!',
+        light: 'red',
+      },
+      'status-warning': 'orange',
+      'status-ok': 'green',
+      'status-unknown': {
+        dark: '#4F5F76',
+        light: '#CCCCCC',
+      },
       'status-disabled': '#CCCCCC',
       blue: {
-        dark: '#0E5265',
+        dark: '#00567A',
         light: '#00C8FF',
       },
       'blue!': '#00739D',
       green: {
-        dark: '#007A5E',
-        light: '#6FFFB0',
+        dark: '#008567',
+        light: '#17EBA0',
       },
       'green!': '#01A982',
       teal: {
-        dark: '#007366',
+        dark: '#117B82',
         light: '#82FFF2',
       },
       'teal!': '#00E8CF',
       purple: {
-        dark: '#371177',
+        dark: '#6633BC',
         light: '#F740FF',
       },
       'purple!': '#7630EA',
       red: {
-        dark: '#4B1916',
-        light: '#FF4F4F',
+        dark: '#A2423D',
+        light: '#FC6161',
       },
-      'red!': '#FF0000',
+      'red!': '#C54E4B',
       orange: {
-        dark: '#CC4B00',
-        light: '#FFB024',
+        dark: '#9B6310',
+        light: '#FFBC44',
       },
       'orange!': '#FF8300',
       yellow: {
-        dark: '#D78F00',
-        light: '#FFEB59',
+        dark: '#8D741C',
+        light: '#FFD63E',
```

Comment:
Looks like this has shifted: https://www.figma.com/file/eZYR3dtWdb9U90QvJ7p3T9/HPE-Color-Styles?node-id=590%3A1231
```suggestion
        light: '#FFEB59',
```